assets/src/scss/base/_mixins.scss

```diff
@@ -40,6 +40,14 @@
   }
 }
 
+// Very large Desktop & Up (greater than 1400px)
+// Used the xxl value by Bootstrap
+@mixin bs-xx-large-and-up {
```

Comment:
Houssam asked me to use the [XXL Bootstrap's breakpoint](https://getbootstrap.com/docs/5.0/layout/breakpoints/#available-breakpoints) instead of the [XXL P4's breakpoint](https://github.com/greenpeace/planet4-master-theme/blob/master/assets/src/scss/base/_variables.scss#L11).

Reply:
If we use that instead of ours, shouldn't we delete `xx-large-and-up`? Even better we should adjust `$extra-extra-large-width` to be `1400px` instead of `1600px`

Reply:
It's a good question, but Houssam requested only in this case to use the XXL from Bootstrap